default-plugins/status-bar/src/main.rs

```diff
@@ -335,7 +335,18 @@ pub fn action_key(keymap: &[(Key, Vec<Action>)], action: &[Action]) -> Vec<Key>
     keymap
         .iter()
         .filter_map(|(key, acvec)| {
-            if acvec.as_slice() == action {
+            if acvec.len() == action.len() {
+                // TODO: is there be a better way?
```

Comment:
How about if we implement something like `Action::is_comparable(&other_action)`?

Reply:
That's a great idea! I'll try to implement it!